test/cypress/features/send.feature

```diff
@@ -9,6 +9,7 @@ Feature: Send
     And I go to transfer confirmation
     And I confirm transfer
     And I go back to wallet
+    And I dismiss the onboarding
```

Comment:
💡 I would consider hiding the onboarding by setting `walletOnboarding = true` to the `localStorage` in the cypress setup:
https://github.com/LiskHQ/lisk-desktop/blob/adafda8fd0cfd73cdbb36e8643cb0f3569a020be/test/cypress/support/commands.js#L36-L40

to keep the feature file simpler. Because without knowing the problem we had today, I don't really see why this step is needed.

Reply:
I agree but Yashar has a different vision on e2e tests